### ISSUE TITLE
Option to ignore specific AssemblyInfo options

### ISSUE THREAD

**Issue by jnyrup** [hvanbakel/CsprojToVs2017#231]
A default `AssemblyInfo.cs` has non-empty assembly options `AssemblyTitle`, `AssemblyProduct` and `AssemblyCopyright`.
When converting projects, I would like to avoid these options being copied to the new csproj.
How about a flag to specify AssemblyInfo.cs options that should not be copied?

E.g.
```
--ignore-assembly-option AssemblyTitle --ignore-assembly-option AssemblyProduct
```

**Comment by hvanbakel** [comment]
That feels verbose you would not want to type that at the command line I
guess.
Isn’t removing the attributes and having no nodes in the project file the
same?

**Comment by jnyrup** [comment]
Yes, removing the attributes from legacy projects before converting gives the intended csproj file without those nodes.

The idea with this option was to avoid the manual work of removing those attributes.

**Comment by mungojam** [comment]
This was one of the reasons I use the convertor as a library rather than an
.exe. It lets us add new pre and post transforms. I actually use a custom
pre transform to strip out assembly info attributes that I know we don't
use so that they don't end up in the final file

**Comment by mungojam** [comment]
@jnyrup hopefully you will be able to do this using the package version once #243 has been merged.

It is one of the things that I do with my use of the package as it means that I can get the `AssemblyInfo.cs` deleted in most cases.

**Comment by jnyrup** [comment]
@mungojam Sounds awesome, thanks for the heads up.

**Comment by mungojam** [comment]
@jnyrup there is now [a beta version](https://www.nuget.org/packages/Project2015To2017/4.1.0-beta.2) of the package that you can use from your own code to achieve things like this. I've updated the readme with a basic example of using the package.

If you get a chance to try it, please let us know how you get on
